subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/DefaultGradleDistribution.java

```diff
@@ -101,7 +101,7 @@ private boolean worksWith(JavaVersion javaVersion) {
             return javaVersion.compareTo(JavaVersion.VERSION_1_7) >= 0 && javaVersion.compareTo(JavaVersion.VERSION_1_10) <= 0;
         }
 
-        return javaVersion.compareTo(JavaVersion.VERSION_1_8) >= 0 && javaVersion.compareTo(JavaVersion.VERSION_1_10) <= 0;
+        return javaVersion.compareTo(JavaVersion.VERSION_1_8) >= 0 && javaVersion.compareTo(JavaVersion.VERSION_11) <= 0;
```

Comment:
I'd get rid of the upper bound, so we don't accidentally stop running tests when we upgrade JDKs.